doc/classes/TreeItem.xml

```diff
@@ -671,6 +671,19 @@
 				Sets the given column's tooltip text.
 			</description>
 		</method>
+		<method name="set_collapsed_recursive">
+			<return type="void" />
+			<argument index="0" name="collapsed" type="bool" />
+			<description>
+				Collapses or uncollapses this TreeItem and all the descendants of this item.
+			</description>
+		</method>
+		<method name="is_any_collapsed">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if this TreeItem, or any of its descendants, are collapsed.
```

Comment:
I noticed the spelling here and will change it to Returns [code]true[/code] if this TreeItem, or any of its descendants, is collapsed.